### ISSUE TITLE
Enable Networking

### ISSUE THREAD

**Issue by cmercur2** [cs-education/sysbuild#161]
The current deployment of the project does not have networking enabled. It is currently supported in the version of jor1k that we are running.

This should have been an easy addition by adding 
`relayURL: 'wss://relay.widgetry.org/'` 
to the Jor1k parameters, but it seems that the state of [sysassets](https://github.com/cs-education/sysassets/) specifically the VM file system contents could be causing the issue. More investigation is needed to confirm.

**Comment by neelabhg** [comment]
Why does networking need to be enabled? If you do need to enable it, I don't think that the `wss://relay.widgetry.org/` websocket URL is meant for production use. We'd need to host our own websocket server.

**Comment by cmercur2** [comment]
Yes, we would need to host our own websocket server. However more work, on top of hosting our own server, must be done to enable networking since adding that parameter is not currently working in our staging deployment.
